src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String essentialText = text.replaceAll("[^a-zA-Z]", "");
+        String reversedEssentialText = new StringBuffer(essentialText).reverse().toString();
```

Comment:
what is the difference between StringBuilder and StringBuffer? What is better to use here?